com.unity.shadergraph/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed treatment of node precision in subgraphs, now allows subgraphs to switch precisions based on the subgraph node [1304050] (https://issuetracker.unity3d.com/issues/precision-errors-when-theres-a-precision-discrepancy-between-subgraphs-and-parent-graphs)
 - Fixed an issue where the Rectangle Node could lose detail at a distance.  New control offers additional method that preserves detail better [1156801]
 - Fixed virtual texture layer reference names allowing invalid characters [1304146]
+- Adjusted doc for keywords to reflect changes from DOC-2830
```

Comment:
Putting your Jira number here doesn't help users - should either be a link to an IssueTracker ticket (see the "Fixed treatment of node precision" entry above) or just a description of the change.

Reply:
I have now revised my note in the Changelog. Thanks Marc!